CHANGELOG.rst

```diff
@@ -14,6 +14,8 @@ Prelude
 
 Release on: 2019-11-27
 
+This release was published for Windows on 2019-12-09.
+This release is not available on MacOS.
```

Comment:
I wouldn't mention MacOS now (we can discuss later if we should always mention macOS or not when it's not released)